### ISSUE TITLE
Chapter 5 training code question/error ?

### ISSUE THREAD

**Issue by wvalcke** [PacktPublishing/PyTorch-Computer-Vision-Cookbook#4]
The output of the Yolo layers use transform_outputs method to convert the box predictions into pixels.
When training we need to revert this back to the original network ouput of the yolo layer. To do this the method transform_bbox is used. It is indeed reverting the operations of transform_outputs, except the last thing done in transform_outputs where the boxes are scaled by the stride factor (32, 16 or 8 depending of the layer). When calculating the loss , we compare directly the calculated targets (via get_yolo_targets) and the 'reverted' yolo network outputs. To me this looks wrong as the reverted yolo outputs still have the stride factor in.

**Comment by mave5** [comment]
thanks for your comment. I will take a look asap.
